data/systemd/meson.build

```diff
@@ -1,5 +1,5 @@
-systemd_system_unit_dir = systemd.get_variable(pkgconfig: 'systemd_system_unit_dir')
-systemd_user_unit_dir = systemd.get_variable(pkgconfig: 'systemd_user_unit_dir')
+systemd_system_unit_dir = systemd.get_variable(pkgconfig: 'systemdsystemunitdir')
+systemd_user_unit_dir = systemd.get_variable(pkgconfig: 'systemduserunitdir')
```

Comment:
Why did you do this change? Is it necessary? It is easier to read `systemd_system_unit_dir` then `systemdsystemunitdir`.